### PR TITLE
Codeclimate fixes

### PR DESCRIPTION
**Pull request by sivagollapalli**

### CODE REVIEW

**Comment by arjun289** [comment]
Please select your base branch as `dev` to be merged with and not `master`. Also, just make sure to rebase your branch with the latest `dev` and squash your commits.

File: lib/gringotts/gateways/paymill.ex [hunk 1]
```diff
@@ -341,7 +344,7 @@ defmodule Gringotts.Gateways.Paymill do
     defp set_success(opts, %{"error" => error}) do
       opts ++ [message: error, success: false]
     end
-    defp set_success(opts, %{"transaction" => %{ "response_code" => 20000}}) do
+    defp set_success(opts, %{"transaction" => %{"response_code" => 20_000}}) do
```
Comment (kartikjagdale) [inline lib/gringotts/gateways/paymill.ex]: I know this follows the style guide. however I want to be skeptical here and I think this unneccesary formating, elixir is ultimatley going to convert this into 20000 so it increase one more processing. Still I would like to know what @ashish173 and @SagarKarwande  thinks over this.

Comment (oyeb) [reply lib/gringotts/gateways/paymill.ex]: IMO it's not unnecessary formatting as it improves readability. Moreover there is no "extra cost" in this as the parser will do the conversion at compile time.

Comment (kartikjagdale) [reply lib/gringotts/gateways/paymill.ex]: @oyeb Thats cool then. lets merge it. 
@ashish173 LGTM :+1:

**Comment by oyeb** [comment]
Fixes #22

**Pull request merged**